ui-tests/src/test/java/org/openmrs/reference/TransferToWardServiceTest.java

```diff
@@ -11,42 +11,41 @@
 
 import org.junit.After;
 import org.junit.Before;
-import org.junit.Ignore;
 import org.junit.Test;
-import org.openmrs.reference.page.ClinicianFacingPatientDashboardPage;
+import org.junit.experimental.categories.Category;
+import org.openmrs.reference.groups.BuildTests;
+import org.openmrs.reference.page.FindPatientPage;
+import org.openmrs.reference.page.PatientVisitsDashboardPage;
 import org.openmrs.uitestframework.test.TestData;
 
-import static org.junit.Assert.assertTrue;
-
+import static org.hamcrest.CoreMatchers.is;
+import static org.hamcrest.MatcherAssert.assertThat;
 
 public class TransferToWardServiceTest extends ReferenceApplicationTestBase {
 
     private static final String INPATIENT_WARD = "Inpatient Ward";
-
     private static final String ISOLATION_WARD = "Isolation Ward";
-
     private TestData.PatientInfo testPatient;
 
     @Before
     public void createTestData() throws Exception {
         testPatient = createTestPatient();
-        String visitType = TestData.getAVisitType();
-        String visit = new TestData.TestVisit(testPatient.uuid, visitType, getLocationUuid(homePage)).create();
     }
 
-    @Ignore("TODO fix, inpatient_ward doesn't exisit on the page, fix workflow")
     @Test
-    public void transferToWardServiceTest() throws Exception {
-
-        ClinicianFacingPatientDashboardPage patientDashboardPage = homePage.goToActiveVisitPatient();
+    @Category(BuildTests.class)
+    public void transferToWardServiceTest() {
+        FindPatientPage findPatientPage = homePage.goToFindPatientRecord();
+        findPatientPage.enterPatient(testPatient.identifier);
+        findPatientPage.waitForPageToLoad();
+        PatientVisitsDashboardPage patientVisitsDashboardPage = findPatientPage.clickOnFirstPatient().startVisit();
 
-        patientDashboardPage.goToAdmitToInpatient().confirm(INPATIENT_WARD);
-        patientDashboardPage.waitForPage();
+        patientVisitsDashboardPage.goToAdmitToInpatient().confirm(INPATIENT_WARD);
 
-        patientDashboardPage.goToTransferToWardServicePage().confirm(ISOLATION_WARD);
-        patientDashboardPage.waitForPage();
+        patientVisitsDashboardPage.goToTransferToWardServicePage().confirm(ISOLATION_WARD);
+        patientVisitsDashboardPage.waitForPage();
 
-        assertTrue(patientDashboardPage.containsText("Outpatient"));
```

Comment:
Refactored the `assertion` to rather validate the encounters than affirming the presence of the text `Outpatient `on the page.